content/en/observability_pipelines/production_deployment_overview/aggregator_architecture.md

```diff
@@ -0,0 +1,414 @@
+
+1. **Direct archiving**: Route data directly from your sources to your archive. This ensures that data makes it to your archive without risk of being dropped. In addition, this data can be replayed after correcting the processing error.
+
+2. **Failed event routing**: The Observability Pipelines Worker offers failed event routing for users who wish to archive processed data, such as structured and enriched data. Certain Observability Pipelines Worker transforms come with a dropped output that can be connected to a sink for durability and replay.
+
+#### Which strategy is best?
+
+If durability is the most important criteria, use the direct archiving method because it addresses data loss scenarios. Use the failed event routing method, also commonly referred to as a data lake, if you prefer to analyze data in your archive. It has the advantage of using your archive/data lake for long-term analysis. Datadog [Log Archives][2] and AWS Athena are examples of archive storage solutions.
+
+### Handling destination failures
+
+Finally, destination failures refer to the total failure of a downstream destination (for example Elasticsearch). Data loss can be mitigated for issues with the downstream destination by using disk buffers large enough to sustain the outage time. This allows data to durably buffer while the service is down and then drain when the service comes back up. For this reason, disk buffers large enough to hold at least one hour's worth of data are recommended. See [disk sizing](#disk-sizing) for more details.
+
+## High availability
+
+High availability refers to Observability Pipelines Worker remaining available if there are any system issues.
+
+To achieve high durability:
+
+1. Deploy at least two Observability Pipelines Worker instances in each Availability Zone.
+2. Deploy Observability Pipelines Worker in at least two Availability Zones.
+3. Front your Observability Pipelines Worker instances with a load balancer that balances traffic across Observability Pipelines Worker instances. See the horizontal scaling section for more information.
+
+### Mitigating failure scenarios
+
+#### Handling Observability Pipelines Worker process issues
+
+To mitigate a system process issue, distribute Observability Pipelines Worker across multiple nodes and front them with a network load balancer that can redirect traffic to another Observability Pipelines Worker instance as needed. In addition, platform-level automated self-healing should eventually restart the process or replace the node.
+
+#### Mitigating node failures
+
+To mitigate node issues, distribute the Observability Pipelines Worker across multiple nodes and front them with a network load balancer that can redirect traffic to another Observability Pipelines Worker node. In addition, platform-level automated self-healing should eventually replace the node.
+
```

Comment:
These two paragraphs are almost identical. Can we condense them to a single paragraph?